com.giuspen.CherryTree.json

```diff
@@ -0,0 +1,145 @@
+{
+  "app-id": "com.giuspen.CherryTree",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cherrytree",
+  "rename-desktop-file": "cherrytree.desktop",
+  "rename-icon": "cherrytree",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host"
+  ],
+  "cleanup": [
+    "/include",
+    "/share/bash-completion",
+    "/lib/pkgconfig/",
+    "/lib/*.a",
+    "/lib/*.la",
+    "/lib/cairo/*.a",
+    "/lib/cairo/*.la",
+    "/share/doc/",
+    "/man/"
+  ],
+  "modules": [
+    {
+      "name": "cairo",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.cairographics.org/releases/cairo-1.16.0.tar.xz",
+          "sha256": "5e7b29b3f113ef870d1e3ecf8adf21f923396401604bda16d44be45e66052331"
+        }
+      ]
+    },
+    {
+      "name": "gtk2",
```

Comment:
Use the one from shared-modules https://github.com/flathub/shared-modules

Reply:
I don't know how to do that can you suggest the change please?

Reply:
ok I see there's a README sorry I'll have a look